packages/xrpl/src/models/common/index.ts

```diff
@@ -108,6 +108,10 @@ export interface ResponseOnlyTxInfo {
    * The sequence number of the ledger that included this transaction.
    */
   ledger_index?: number
+  /**
+   * @deprecated Alias for ledger_index.
+   */
+  inLedger?: number
```

Comment:
Why is this used in tests instead of `ledger_index`?

Reply:
It should be changed --- fixed

Reply:
bump

Reply:
bump

Reply:
Removed check

Reply:
Can we remove it from the model too in that case?

Reply:
@mvadari it’s still returned by the server

Reply:
Yes, but if it's deprecated we shouldn't support people using it. For example, we don't support the deprecated parts of PathSteps https://xrpl.org/paths.html#path-specifications

Reply:
While I agree, I'd be more annoyed as a user to get back a response from the server that contains a field that isn't in the typescript types. I prefer to see `@deprecated`. Otherwise we could end up with people filing bugs that were missing a type. Likewise there's code like this:

```
export async function verifySubmittedTransaction(
  client: Client,
  tx: Transaction | string,
  hashTx?: string,
): Promise<void> {
  const hash = hashTx ?? hashSignedTx(tx)
  const data = await client.request({
    command: 'tx',
    transaction: hash,
  })

  assert(data.result)
  assert.deepEqual(
    omit(data.result, [
      'date',
      'hash',
      'inLedger',
      'ledger_index',
      'meta',
      'validated',
    ]),
    typeof tx === 'string' ? decode(tx) : tx,
  )
  if (typeof data.result.meta === 'object') {
    assert.strictEqual(data.result.meta.TransactionResult, 'tesSUCCESS')
  } else {
    assert.strictEqual(data.result.meta, 'tesSUCCESS')
  }
}
```

It would be odd to be omitting a type from a result object that doesn't exist in the typescript definition.

Reply:
That's fair, we can leave it alone. In that case, mind adding a followup for the other deprecated types?

Reply:
Added to followup